src/ResourceManager/Dns/AzureRM.Dns.psd1

```diff
@@ -54,7 +54,7 @@ CLRVersion = '4.0'
 RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.3.1'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Management.Dns.dll'
+RequiredAssemblies = '.\Microsoft.Azure.Management.Dns.dll', '.\Microsoft.Azure.Commands.Network.dll'
```

Comment:
@muwaqar this reference to `Microsoft.Azure.Commands.Network.dll` can be removed